forge-gui/res/cardsfolder/upcoming/nemata_primeval_warden.txt

```diff
@@ -0,0 +1,14 @@
+Name:Nemata, Primeval Warden
+ManaCost:2 B G
+Types:Legendary Creature Treefolk
+PT:3/4
+K:Reach
+R:Event$ Moved | ActiveZones$ Battlefield | Origin$ Battlefield | Destination$ Graveyard | ValidLKI$ Card.Creature+nonToken+OppCtrl | CheckSelfLKIZone$ True | ReplaceWith$ Exile | Description$ If a nontoken creature an opponent controls would die, exile it instead. When you do, create a 1/1 green Saproling creature token.
+SVar:Exile:DB$ ChangeZone | Hidden$ True | Origin$ All | Destination$ Exile | Defined$ ReplacedCard | SubAbility$ DBImmediateTrigger
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | Execute$ TrigToken | TriggerDescription$ If a nontoken creature an opponent controls would die, exile it instead. When you do, create a 1/1 green Saproling creature token.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ g_1_1_saproling | TokenOwner$ You
+A:AB$ Pump | Cost$ G Sac<1/Saproling/a Saproling> | NumAtt$ +2 | NumDef$ +2 | AILogic$ Aristocrat | SpellDescription$ CARDNAME gets +2/+2 until end of turn.
+A:AB$ Draw | Cost$ 1 B Sac<2/Saproling/two Saprolings> | Defined$ You | NumCards$ 1
```

Comment:
Did you try these with Sac<1/Saproling> and Sac<2/Saproling>? I thought I fixed the parsing on something like this, but can't remember

Reply:
You're right, looks like the sacrifice prompt looks just fine without the descriptor. Added `Type$Saproling` and a missing `SpellDescription` for the second activated ability